.github/workflows/lint-test.yml

```diff
@@ -0,0 +1,31 @@
+name: Lint, and Test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  lint-test:
+    name: Lint, and Test
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn --frozen-lockfile --ignore-scripts
+      - run: yarn lint
+      - run: yarn test
+  all-jobs-pass:
+    name: All jobs pass
+    runs-on: ubuntu-20.04
+    needs:
+      - lint-test
+    steps:
+      - uses: actions/checkout@v2
+      - run: echo "Great success!"
```

Comment:
Nit: This file is missing a trailing newline

Reply:
fixed